docker-compose.yaml

```diff
@@ -51,8 +49,36 @@ services:
         ipv4_address: 172.25.0.10
     volumes:
       - ./certs:/etc/letsencrypt:rw
-    
-    
+         
+  postgres:
+    container_name: postgres
+    build:
+      context: .
+      dockerfile: Dockerfile.postgres
+      args:
+        CRYPT_PASSWORD: "${CRYPT_PASSWORD}"
+    environment:
+      CRYPT_PASSWORD: "${CRYPT_PASSWORD}"   
+    volumes:
+      - ./pg_data1:/var/lib/postgresql/data
```

Comment:
Lets not use local folders for postgres db files. Instead lets use volumes

lets add local folders  that we mount inside  containers to  `.gitignore` `.dockerignore`.
Folders:
* ./certs
* ./logs

Reply:
I need to create tasks in ansible so that these files are created and added to .gitignore .dockerignore, right?